src/handlers/transfers/FxFulfilService.js

```diff
@@ -141,12 +176,20 @@ class FxFulfilService {
     if (!dupCheckResult.hasDuplicateHash) {
       // ERROR: We've seen fxTransfer of this ID before, but it's message hash doesn't match the previous message hash.
       const fspiopError = fspiopErrorFactory.noFxDuplicateHash()
-      const apiFSPIOPError = fspiopError.toApiErrorObject(this.Config.ERROR_HANDLING)
+      const apiFSPIOPError = fspiopError.toApiErrorObject(
+        this.Config.ERROR_HANDLING
```

Comment:
Is it necessary to reformat like this?

Reply:
@geka-evk Could be the linter in the pre-commit hook. Will fix that.